File: Ex14.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    // Inserção das variáveis

    int NumeroX, NumeroY, NumeroZ;

    // Inserção do primeiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroX);

    
    // Inserção do segundo valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroY);
    
    
    // Inserção do terceiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroZ);

    //Comparação entre valores
    if (NumeroX>NumeroY)
    {
        if (NumeroX>NumeroZ)
        {
            printf("O maior valor inserido e %i", NumeroX);
        }
        else {
            printf("O maior valor inserido e %i", NumeroZ);
        }
    }
    else{
    
        if (NumeroY>NumeroZ)
        {
            printf("O maior valor inserido e %i", NumeroY);
        }
        else {
            printf("O maior valor inserido e %i", NumeroZ);
        }
        

    }
        
    
    
    
    return 0;
}

File: Ex14.py
v1 = float(input("Insira um numero qualquer: "))
v2 = float(input("Insira um outro numero qualquer: "))
v3 = float(input("Insira um terceiro numero qualquer: "))

if v1 < v2:

    if v2 < v3:
        print(f"O número {v3:} e o maior entre os tres.")

    else:
        print(f"O número {v2:} e o maior entre os tres.")


else:
    print(f"O número {v1:} e o maior entre os tres.")